windows-installer/Package.wxs

```diff
@@ -52,7 +51,7 @@
     <!-- Execution of uninstall command-->
     <SetProperty
       Id="RunUninstallAction"
-			Value="&quot;[%SystemFolder]cmd.exe&quot; /c &quot;%IDE_ROOT%\_ide\installation\bin\ideasy.exe -f uninstall&quot;"
+			Value="&quot;[%SystemFolder]cmd.exe&quot; /c &quot;[%IDE_ROOT]\_ide\installation\bin\ideasy.exe -f uninstall&quot;"
```

Comment:
`ideasy.exe -f uninstall` still not working. The msi log only return `1`. But since we ignore the return for now, we dont have deadlocks anymore

Reply:
For the record: For debugging we can use `&quot;«command»&quot; > «abs-path-to-log»` and then check the log.
Then we can verify if IDEasy was actually found and invoked or not and if invoked, what the output and error was.

Reply:
All fine now, I set this to resolved. Thanks.